options.js

```diff
@@ -17,26 +17,26 @@ function save_options() {
 function clear_options() {
     chrome.storage.sync.remove("WKapikey");
     chrome.storage.sync.remove("notifyWKlessons");
-    
+
     let alert = document.getElementById('alert');
     alert.innerHTML = 'All cleared! - Now add your API token';
 }
 
 document.getElementById('save').addEventListener('click', save_options);
 document.getElementById('clear').addEventListener('click', clear_options);
 
-// retrieve current status and updates checkbok
-function update_checkbox() {
-    chrome.storage.sync.get("notifyWKlessons", function(result) {
-        var lessonsenabled = result.notifyWKlessons;
-
+// Retrieve options from chrome.storage
+function load_options() {
+    chrome.storage.sync.get(["notifyWKlessons", "WKapikey"], function(result) {
+        document.getElementById('apikey').value = result.WKapikey;
```

Comment:
I don't really like to unnecessarily expose the API key through the UI.
Can you undo this change?

Reply:
So the API is already "exposed" through the WK website, and this settings page is entirely local. Leaving it blank when an API key has already been entered is confusing because it appears that it needs to be entered again. Due to the minute security risks, I believe that the UX is more important here. Let me know if you still disagree though.

Reply:
That's fair, I was thinking of a situation where you'd install the extension on a shared computer but I appreciate that's not a very realistic use case now that you mention it, so happy to implement this.